.storybook/parts/default-categories.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Manualy populate categories for blocks. This is generated in the PHP part of the real project.
+ */
+wp.data.dispatch( 'core/blocks').setCategories([
+  {
+    slug: 'eightshift',
+    title: 'Eightshift',
+    'icon': 'admin-settings',
```

Comment:
```suggestion
    icon: 'admin-settings',
```

This doesn't have to be quoted?